src/test/java/com/sngular/kloadgen/extractor/extractors/AvroExtractorTest.java

```diff
@@ -22,9 +22,8 @@ void testFlatPropertiesEmbeddedAvros() throws Exception {
 
```

Comment:
Please add test to check the behavior of the new extractor interface

Reply:
the interface?

Reply:
Yes, when I refer to the interface on that line it is for the methods related to the new `Extractor` interface instead of call the missing test methods

Reply:
Do we have any input example? because that code hadn't got test before the refactor, and I have no idea of the input needed.